.changeset/fifty-bobcats-beg.md

```diff
@@ -0,0 +1,5 @@
+---
+'@directus/schema': patch
+---
+
+Fixed an issue that would cause directus to crash on unexpected database schema contents
```

Comment:
(I'd maybe mention that it only concerns PostgreSQL databases

```suggestion
Fixed an issue that would cause a server crash on unexpected PostgreSQL database schema contents
```

)